src/shader_recompiler/ir/passes/hull_shader_transform.cpp

```diff
@@ -386,9 +387,16 @@ void HullShaderTransform(IR::Program& program, RuntimeInfo& runtime_info) {
                     return ir.BitCast<IR::F32, IR::U32>(IR::U32{data});
                 };
                 const u32 num_dwords = u32(opcode) - u32(IR::Opcode::StoreBufferU32) + 1;
-                IR::U32 index = IR::U32{inst.Arg(1)};
-                ASSERT(index.IsImmediate());
-                const u32 gcn_factor_idx = (info.inst_offset.Value() + index.U32()) >> 2;
+                IR::Value voffset, soffset;
+                bool success =
+                    M_COMPOSITECONSTRUCTU32X3(MatchU32(0), MatchImm(voffset), MatchValue(soffset))
+                        .Match(inst.Arg(IR::StoreBufferArgs::Address));
+                // Some shaders seen using an uninitialized sgpr for soffset. Ignore undef soffset
+                // arg. TODO figure out if there's defined behavior or just UB
```

Comment:
That doesn't sound possible for compiler to emit, there must be some initialization SGPR missing